services/provider/server/server.go

```diff
@@ -650,6 +667,23 @@ func (s *OCSProviderServer) appendStorageClassExternalResources(
 			drCephFsId,
 		)
 	}
+	scMap[util.GenerateNameForCephNetworkFilesystemSC(storageCluster)] = func() *storagev1.StorageClass {
+		return util.NewDefaultNFSStorageClass(
+			consumerConfig.GetNfsClientProfileName(),
+			util.GenerateNameForCephNFS(storageCluster),
+			util.GenerateNameForCephFilesystem(storageCluster.Name),
+			util.GenerateNameForNFSService(storageCluster),
+			consumerConfig.GetCsiNfsProvisionerSecretName(),
+			consumerConfig.GetCsiNfsNodeSecretName(),
+			consumer.Status.Client.OperatorNamespace,
+		)
+	}
+	scMap[util.GenerateNameForCephRgwSC(storageCluster)] = func() *storagev1.StorageClass {
+		return util.NewDefaultOBCStorageClass(
+			consumer.Status.Client.OperatorNamespace,
+			util.GenerateNameForCephObjectStore(storageCluster),
+		)
+	}
```

Comment:
This should not be here. It has nothing to do with our block and FS CSI provisioners